domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/brevmapper/brev/InnvilgelseEngangstønadDokumentdataMapper.java

```diff
@@ -55,14 +55,15 @@ public String getTemplateNavn() {
                 .erAutomatiskBehandlet(dokumentFelles.getAutomatiskBehandlet())
                 .harVerge(dokumentFelles.getErKopi().isPresent())
                 .erKopi(dokumentFelles.getErKopi().isPresent() && erKopi(dokumentFelles.getErKopi().get()))
+                .saksnummer(dokumentFelles.getSaksnummer().getVerdi())
                 .build();
 
         var innvilgelseDokumentDataBuilder = EngangsstønadInnvilgelseDokumentdata.ny()
                 .medFelles(felles)
                 .medRevurdering(behandling.erRevurdering())
                 .medFørstegangsbehandling(behandling.erFørstegangssøknad())
                 .medMedhold(BehandlingMapper.erMedhold(behandling))
-                .medInnvilgetBeløp(beregningsresultat.getBeløp())
+                .medInnvilgetBeløp(String.format("%,d",beregningsresultat.getBeløp()))
```

Comment:
Trekker du ut formateringen i en hjelpemetode som beskriver hva som skjer i navnet? Så slipper du å skrive den samme koden to steder

Reply:
utført